kroxylicious-api/src/main/java/io/kroxylicious/proxy/config/tls/TlsClientAuth.java

```diff
@@ -0,0 +1,25 @@
+package io.kroxylicious.proxy.config.tls;
```

Comment:
Missing licence

Reply:
Hmmmm that shouldn't be an issue the maven build should be adding them automatically...